lib/request_handler/concerns/config_helper.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module RequestHandler
+  module Concerns
+    module ConfigHelper
+      def lookup!(hash, key)
```

Comment:
can't you reuse the `lookup` method inside here instead of duplicating the code?